test/e2e/scheduling/resource_quota.go

```diff
@@ -365,6 +365,22 @@ var _ = SIGDescribe("ResourceQuota", func() {
 	})
 
 	It("should create a ResourceQuota and capture the life of a configMap.", func() {
+		found, unchanged := 0, 0
```

Comment:
Did this test begin to fail? As above, I don't think this is the right suite to test the ca.crt configmap publisher. It doesn't roll up to the right owners if this suite starts to fail. Something in sig-auth would be better.